src/controllers/address.ts

```diff
@@ -1,23 +1,25 @@
-import { Request, Response, Router } from "express";
-import { handlerWrapper } from "../errors/AppError";
-import { convertToHexIfBech32, validateAddress } from "../helpers/validator";
-import { fetchFaucetBalance } from "../repository/address";
+import { Request, Response, Router } from 'express'
+import { handlerWrapper } from '../errors/AppError'
+import { convertToHexIfBech32, validateAddress } from '../helpers/validator'
+import { fetchFaucetBalance } from '../repository/address'
 
-const router = Router();
+const router = Router()
 
 const getFaucetBalance = async (req: Request, res: Response): Promise<any> => {
-  let address = convertToHexIfBech32(req.query.address as string)
-  if (validateAddress(address)){
-    address = address.length === 56 ? `e0${address}`:address
-  }else{
-    return res.status(400).json({message:'Provide a valid address'})
-  }
+    let address = convertToHexIfBech32(req.query.address as string)
+    if (validateAddress(address)) {
```

Comment:
Fix the validateAddress function instead. What is the use of `validateAddress` function if returns `true` for invalid address